easybuild/easyblocks/b/boost.py

```diff
@@ -111,11 +113,15 @@ def configure_step(self):
             else:
                 raise EasyBuildError("Unknown compiler used, don't know what to specify to --with-toolset, aborting.")
 
-        cmd = "./bootstrap.sh --with-toolset=%s --prefix=%s %s" % (toolset, self.objdir, self.cfg['configopts'])
+        configopts += " --with-toolset=%s" % toolset
+
+        if not self.cfg['boost_mpi']:
+            configopts += " --without-libraries=mpi"
+
+        cmd = "./bootstrap.sh %s %s" % (configopts, self.cfg['configopts'])
```

Comment:
let's use `preconfigopts` too, while we're at it?

```python
cmd = "%s ./bootstrap.sh %s %s" % (self.cfg['preconfigopts'], configopts, self.cfg['configopts'])
```

or (whatever looks nicer to you)

```python
cmd = ' '.join([self.cfg['preconfigopts'], 'bootstrap.sh', self.cfg['configopts']])
```